recipes/underworld/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "underworld" %}
+{% set version = "2.14.1" %}
+{% set sha256 = "ec922eb6d10d6a6031d5c76b106df0c79b122ee61c43900866f2017bf819b2eb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ad/44/e491e7faa66cd927b722304d3e0a7ecc5f6c24cefa6044868a84548303d5/underworld-{{ version }}b0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
+  number: 0
+  script: PETSC_DIR=$PREFIX {{ PYTHON }} -m pip -vv install --no-deps .  
+  # shared objects to ignore when checking linkage information
+  missing_dso_whitelist:
+    - "**/site-packages/underworld/lib/**"
+
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ mpi }}
+    - petsc {{ petsc }}
+    - pkg-config
+    - cmake
+    - swig >=4.0.0
+
+  host:
+    - pip
+    - setuptools
+    - python
+    - libxml2
+    - numpy >=1.18
+    - ninja
+    - petsc {{ petsc }}
+    - {{ mpi }}
+
+  run:
+    - python
+    - {{ mpi }}
+    - numpy
+    - petsc {{ petsc }}
+    - pint
+    - scipy
+    - mpi4py
+    - h5py
+
+test:
+  imports:
+    - underworld
+
+  requires:
+    - pytest
+    - matplotlib
+
+      #  source_files:
+      #    - docs/pytests/test.py
+      #    - docs/pytests/test_examples.py
+
+  commands:
+    - python -X faulthandler -c "import underworld; underworld.mesh.FeMesh_Cartesian();"
+      #- pytest -vvv docs/pytests/*
+
+
+about:
+  home: https://github.com/underworldcode/underworld2
+  license: LGPL-3.0-only
+  license_family: LGPL
+  license_file: LGPLv3.txt
+  summary: "Geodynamics API"
+  doc_url: https://underworld2.readthedocs.io
+  dev_url: https://github.com/underworldcode/underworld2/tree/development
+
+extra:
+  recipe-maintainers:
+    - julesghub
+    - lmoresi
+    - rbeucher
```

Comment:
can you please ask these two to affirm they are willing to maintain.

Reply:
Removed these two